### PR TITLE
Add req.ws to typedefs

### PR DESCRIPTION
**Pull request by a-a-GiTHuB-a-a**
Wow, what a dumb oversight I made with my original pull request.

I use the GitHub website, so I have absolutely no idea how to run tests, but I checked and it looks okay.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -23,6 +23,10 @@ declare module 'fastify' {
     websocketServer: WebSocket.Server,
   }
 
+  interface FastifyReply {
```
Comment (darkgl0w) [inline index.d.ts:26]:
```suggestion
  interface FastifyRequest {
```

**Review by darkgl0w — CHANGES_REQUESTED** [review]
Can you add a unit test please ?

**Comment by a-a-GiTHuB-a-a** [comment]
Oops, I'm dumb.

**Comment by airhorns** [comment]
Thanks for the contribution, shipped in #188

**Pull request closed without merging**